### DISCUSSION TITLE
Support for MAUI Blazor App

### DISCUSSION THREAD

**Discussion by SvenVoeten** [vikramlearning/blazorbootstrap#309] [Q&A] [answered]
Hi All,

Is it possible to use this library with an .NET MAUI Blazor app?
I did get it working for the Blazor WebAssebly app but the simular approach doesn't work for MAUI.
Am I doing something wrong?

Eventualy I want to develop an android/windows app and bootstrap seems to have a nice kit for the UI.

**Comment by gvreddy04** [comment]
@SvenVoeten It will work for MAUI as well. At this moment, I don't have a sample, but I will create some documentation for it.

**Reply by SvenVoeten** [reply]
@gvreddy04 

Thanks, looking forward to it.

**Comment by aldvsmaximvs** [comment, marked as answer]
Hi @SvenVoeten 

It does work by following the exact instructions specified in the documentation and register the service on MauiProgram.cs

**Reply by SvenVoeten** [reply]
@aldvsmaximvs 

I've come to the same conclusion but thanks for testing.
The only problem I still have is that I get an "unhandled error" but my app does work.
Did you also get this error?

![image](https://github.com/vikramlearning/blazorbootstrap/assets/65830467/34c0696c-67c1-44fa-a895-faf2bb91df7b)
